pymeasure/instruments/attocube/anc300.py

```diff
@@ -255,3 +247,65 @@ def stop_all(self):
             attribute = getattr(self, attr)
             if isinstance(attribute, Axis):
                 attribute.stop()
+
+    def __init__(self, host, axisnames, passwd, query_delay=0.05, **kwargs):
+        kwargs['query_delay'] = query_delay
```

Comment:
Giving `query_delay` to the adapter is deprecated, as we implement that waiting in the instrument itself.
so, store it in `self.query_delay = query_delay` and implement a function (that way you have that delay per default):
~~~python
    def wait_for(self, query_delay=0):
        """Wait for some time. Used by 'ask' to wait before reading.

        :param query_delay: Delay between writing and reading in seconds.
        """
        super().wait_for(query_delay or self.query_delay)
~~~

Reply:
Added this in, thanks for the method.